auth-api/requirements.txt

```diff
@@ -41,21 +41,22 @@ marshmallow==3.18.0
 minio==7.1.12
 opentracing==2.4.0
 packaging==21.3
+pkg_resources==0.0.0
```

Comment:
what happens if you take this line out?

Reply:
nothing happens when I remove it. pkg_resources==0.0.0 keeps popping back up in requirements whenever I run pip install, it's pretty annoying, especially when I forget that I ran pip install before making a commit.

Reply:
instead of pip install you should be able to just run 'make setup' so that you don't update the reqs which can cause other parts of the app to fail out if there's a dependency issue with one of the new versions

Reply:
removed pkg_resources==0.0.0